server/dump.go

```diff
@@ -56,17 +59,12 @@ func (p *Plugin) createDump(ctx context.Context, min, max time.Time, remoteStora
 		return "", err
 	}
 
-	err = db.CleanTombstones()
```

Comment:
Why did this get removed?

Reply:
That step turned out to be redundant as there were no tombstones were set before. Even if there is, compaction handles that.